enhancements/metering/user-provided-pricesheets.md

```diff
@@ -0,0 +1,205 @@
+---
+title: user-provided-price-sheets
+authors:
+  - "@chancez"
+reviewers:
+  - @tflannag
+  - "@emilym1"
+approvers:
+  - "@operator-framework/team-metering"
+creation-date: 2019-11-06
+last-updated: 2019-11-06
+status: implementable
+---
+
+# user-provided-price-sheets
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+TBD
+
+## Summary
+
+User provided price sheets is a way for users to define custom price rates for calculating cost reports on resource reservations or usage.
+It is an opt-in feature that would require the user to provide configuration that defines the prices and rates they desire.
+
+## Motivation
+
+Currently metering only supports calculating costs using AWS billing data, but does not support calculating costs for other clouds nor on-premise solutions.
+User provided price sheets would allow users to define their own prices for uses in clouds or environments which don't yet have native support from Metering.
+
```

Comment:
User provided price sheets would enable Metering to expand it's functionality to those who wish to define their own prices for clouds and environments which are not yet supported.